src/entity/Unit.ts

```diff
@@ -184,6 +185,7 @@ export function create(
       staminaMax,
       attackRange: config.UNIT_BASE_RANGE,
       isMiniboss: false,
+      isBoss: false,
```

Comment:
I'd rather just check for the unit id if we need to determine "boss-ness" rather than adding a property to every unit.  Remember every property that we add to units needs to be send over the network very frequently for every unit unless it's optional.